ui/src/reusable_ui/components/radio_buttons/RadioButtons.tsx

```diff
@@ -35,19 +36,28 @@ class RadioButtons extends Component<Props> {
         className={this.buttonClassName(button)}
         onClick={this.handleButtonClick(button)}
       >
-        {button}
+        {this.buttonText(button)}
       </div>
     ))
   }
 
+  private buttonText = (button): JSX.Element => {
+    if (_.startsWith(button, 'icon')) {
```

Comment:
I wonder if there's a more widely reusable API for a component like this. Maybe something like
```
<RadioButtons activeKey={"play"} onSelect={this.handleSelect} />
  <RadioButton key="play">
    <span className="icon-this" />
  </RadioButton>
  <RadioButton key="pause">
    <span className="icon-that" />
  </RadioButton>
</RadioButtons>
```
not sure, just spitballing

Reply:
I like this idea -- do you know how I would tell each `RadioButton` to change its class in response to being active? Like how would the radio buttons get information from their parent?